Sdk/Sdk.targets

```diff
@@ -1,24 +1,18 @@
-<Project>
+<Project InitialTargets="ResolveElskomSharedFramework">
 
   <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" Condition="'$(ElskomSdkMicrosoftNETSdkImported)' == 'true'" />
 
-  <ItemGroup>
-    <!-- The libraries within the runtime pack are within the Elskom/runtime repository. -->
-    <KnownFrameworkReference Include="Elskom.Sdk.App"
-                             TargetFramework="net6.0"
-                             RuntimeFrameworkName="Elskom.Sdk.App"
-                             DefaultRuntimeFrameworkVersion="$(ElskomSdkFrameworkVersion)"
-                             LatestRuntimeFrameworkVersion="$(ElskomSdkFrameworkVersion)"
-                             TargetingPackName="Elskom.Sdk.App.Ref"
-                             TargetingPackVersion="$(ElskomSdkFrameworkVersion)"
-                             RuntimePackNamePatterns="Elskom.Sdk.App.Runtime.**RID**"
-                             RuntimePackRuntimeIdentifiers="win-x86;win-x64;win-arm64;linux-x64;linux-arm;linux-arm64;osx-x64;osx-arm64"
-                             IsTrimmable="true"
-                             Condition="'$(ElskomSdkFrameworkVersion)' != ''" />
-    <FrameworkReference
-            Include="Elskom.Sdk.App"
-            IsImplicitlyDefined="true"
-            Condition="'$(DisableImplicitFrameworkReferences)' != 'true' AND ('$(ElskomSdkDisableImplicitFrameworkReference)' != 'true' OR '$(ElskomSdkDisableImplicitFrameworkReference)' == '')" />
-  </ItemGroup>
+  <UsingTask
+             TaskName="Elskom.Sdk.Tasks.AddElskomSdkFrameworkReference"
+             AssemblyFile="$(MSBuildThisFileDirectory)/../tasks/netstandard2.0/Elskom.Sdk.Tasks.dll" />
+
+  <Target Name="ResolveElskomSharedFramework">
+    <AddElskomSdkFrameworkReference
+      DisableImplicitReference="$(ElskomSdkDisableImplicitFrameworkReference)">
+      <!-- The libraries within the runtime pack are within the Elskom/runtime repository. -->
+      <Output TaskParameter="KnownFrameworkReferences" ItemName="KnownFrameworkReference" />
+      <Output TaskParameter="FrameworkReferences" ItemName="FrameworkReference" />
+    </AddElskomSdkFrameworkReference>
+  </Target>

```

Comment:
Despite trying to get this to run, it seems to never really add the ``FrameworkReference`` for some reason, also any projects using the sdk seems to never fully resolve inside of Visual Studio.